src/components/collapsible.tsx

```diff
@@ -39,13 +43,20 @@ const Collapsible: FC<Props> = ({
   opacityOnHover = true,
   forceCollapse = true,
   alwaysRender = false,
+  tealChevron = false,
+  mobileChevron = false,
 }) => {
   const [isCollapsed, setIsCollapsed] = useState(forceCollapse)
 
   useEffect(() => {
     setIsCollapsed(forceCollapse)
   }, [forceCollapse])
 
+  const arrowColor = tealChevron ? "text-teal" : ""
```

Comment:
I would ask the designer to opt for a consistent color instead of introducing switches like this. it's pretty likely they did not notice that they're introducing another variant